api/src/main/java/org/pmiops/workbench/db/model/DbParticipantCohortStatus.java

```diff
@@ -192,25 +147,14 @@ public boolean equals(Object o) {
     DbParticipantCohortStatus that = (DbParticipantCohortStatus) o;
     return status == that.status
         && Objects.equals(genderConceptId, that.genderConceptId)
-        && Objects.equals(gender, that.gender)
         && Objects.equals(birthDate, that.birthDate)
         && Objects.equals(raceConceptId, that.raceConceptId)
-        && Objects.equals(race, that.race)
-        && Objects.equals(ethnicityConceptId, that.ethnicityConceptId)
-        && Objects.equals(ethnicity, that.ethnicity);
+        && Objects.equals(ethnicityConceptId, that.ethnicityConceptId);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(
-        status,
-        genderConceptId,
-        gender,
-        birthDate,
-        raceConceptId,
-        race,
-        ethnicityConceptId,
-        ethnicity);
+    return Objects.hash(status, genderConceptId, birthDate, raceConceptId, ethnicityConceptId);
```

Comment:
I wish we had a linter to find places where we leave out members in this method...